src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -352,9 +335,26 @@ public string Url
             }
         }
 
+        /// <summary>
+        /// The uri to load.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Uri Uri
```

Comment:
Please keep [EditorBrowsable(EditorBrowsableState.Never)] in DevelNUI branch.
When this patch is merged, you can remove it on Master branch with ACR.